src/test/operators/import_csv_test.cpp

```diff
@@ -250,9 +253,10 @@ TEST_F(OperatorsImportCsvTest, WithAndWithoutQuotes) {
 }
 
 TEST_F(OperatorsImportCsvTest, StringDoubleEscape) {
-  auto config = CsvConfig{};
-  config.escape = '\\';
-  auto importer = std::make_shared<ImportCsv>("src/test/csv/string_double_escape.csv", config);
+  std::string csv_file = "src/test/csv/string_double_escape.csv";
+  auto csv_meta = process_csv_meta_file(csv_file + CsvMeta::META_FILE_EXTENSION);
+  csv_meta.config.escape = '\\';
+  auto importer = std::make_shared<ImportCsv>(csv_file, csv_meta);
```

Comment:
Can we get some more tests for invalid configs? Like JSON syntax errors, mismatching column information, invalid options, etc.